Vwitter/View Controllers/SelectAudienceViewController.m

```diff
@@ -0,0 +1,152 @@
+//
+//  SelectAudienceViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "SelectAudienceViewController.h"
+#import <Parse/Parse.h>
+#import "Follow.h"
+#import "AudienceMemberCell.h"
+#import "Vent.h"
+#import "VentAudience.h"
+#import <MBProgressHUD/MBProgressHUD.h>
+
+@interface SelectAudienceViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UIButton *postVentButton;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSMutableArray *arrayOfAudienceMembers;
+@property (strong, nonatomic) NSMutableArray *arrayOfSelectedAudience;
+
+@end
+
+@implementation SelectAudienceViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.arrayOfSelectedAudience = [[NSMutableArray alloc] init];
+    self.tableView.allowsMultipleSelection = YES;
+    [self loadData];
+
+    self.postVentButton.layer.cornerRadius = 20;
+    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
```

Comment:
nit: This is a bit dangerous, because `loadData` could, in principle (though very unlikely) finish before we've even gotten to setting the dataSource & delegate of the table view, and so when it reloads the table view, it won't have anywhere to get data from and nothing would happen, even if you did load some audience.